timescale-forge/disk-management.md

```diff
@@ -0,0 +1,112 @@
+# Disk management
+
+Timescale Forge contains several mechanisms for managing disk space on your
+services. There are four key tasks that Forge performs to handle disk space:
+
+1.  Detect if storage capacity begins to fill up
+1.  Notify you about the growth
+1.  Automatically activate overload protections
+1.  Allow you to return your database to a normal state
+
+This section explains what the various mechanisms are, and how to best make use
+of them.
+
+## Continuous storage monitoring
+
+Timescale Forge continuously monitors the health and resource consumption of all
+database services. You can check your health data by navigating to the `metrics`
+tab in your service dashboard. These metrics are also monitored by the Timescale
+operations team.
+
+<img class="main-content__illustration" src="https://assets.timescale.com/images/diagrams/forge_metrics.png" alt="Timescale Forge metrics dashboard"/>
+
+If your database exceeds a storage threshold of available resources, some
+automated actions are triggered, including notifications, and preventative
+actions.
+
+## Automated user alerting
+
+When your disk usage exceeds certain thresholds, you will recieve email
+notifications. These notifications occur at:
+
+*   75%
+*   85%
+*   95%
+
+So that you aren't overwhelmed by automated message, the alerting thresholds use
+low- and high-watermarks, and only one message is sent per database per day.
```

Comment:
```suggestion
low- and high-watermarks, and only one message is sent per database per threshold per day.
```
Right now it is working like that. (Added changeset just to reflect facts - do not think that text should contain per/per/per... that makes it complicated.)

(my 2 cents:)
But I personally think that this can be changed in the future, and we can just say that number of messages is limited. 
cc @lukas-bernert